src/nxdoc/nuxeo-add-ons/nuxeo-enhanced-viewer/how-to-import-documents-with-annotations.md

```diff
@@ -0,0 +1,537 @@
+        // arenderAnnotation.setInreplyto(new AnnotationId(...));
+        arenderAnnotation.setCreator(creator);
+        arenderAnnotation.setCreationDate(creationDate);
+        arenderAnnotation.setDate(modificationDate);
+        // then use appropriate setter depending on the annotation type you choose
+        // ....
+
+        // then serialize the annotation
+        org.nuxeo.client.objects.comment.Annotation nuxeoAnnotation = new org.nuxeo.client.objects.comment.Annotation();
+        nuxeoAnnotation.setEntityId(new AnnotationIdAdapter().marshal(arenderAnnotation.getId()));
+        nuxeoAnnotation.setOrigin("ARender"); // has to be ARender
+        try (InputStream stream = annotationConverter.serialize(arenderDocumentId, singletonList(arenderAnnotation))) {
+            nuxeoAnnotation.setEntity(IOUtils.toString(stream, UTF_8));
+        }
+        nuxeoAnnotation.setXpath("file:content");
+        nuxeoAnnotation.setAuthor("jdoe"); // user id
+        nuxeoAnnotation.setCreationDate(creationDate.toInstant());
+        nuxeoAnnotation.setModificationDate(modificationDate.toInstant());
+        // if the annotation is a TextualAnnotation, you can set the text in order to display it into Nuxeo
+        nuxeoAnnotation.setText(((TextualAnnotation) arenderAnnotation).getContents());
+
+        // then save it to Nuxeo
+        client.repository().newDocumentAdapter(nuxeoDocId, AnnotationAdapter::new).create(nuxeoAnnotation);
+
+    }
+
+    public static class NuxeoDocumentService extends DocumentServiceDelegate {
+
+        private final DocumentLayout layout;
+
+        protected NuxeoDocumentService(DocumentLayout layout) {
+            super(true);
+            this.layout = layout;
+        }
+
+        @Override
+        public DocumentLayout getDocumentLayout(DocumentId documentId) {
+            return layout;
+        }
+
+        @Override
+        public void loadDocumentAccessor(DocumentAccessor documentAccessor,
+                DocumentAccessorSelector documentAccessorSelector) {
+            // not needed
+        }
+
+        @Override
+        public DocumentService getDelegate() throws DocumentServiceDelegateNotAvailable {
+            // not needed
+            return null;
+        }
+
+        @Override
+        protected void checkDocumentAvailability(DocumentId documentId) {
+            // not needed
+
+        }
+
+        @Override
+        public DocumentId mayProcessAnnotationRendition(DocumentId documentId) {
+            // not needed
+            return null;
+        }
+
+        @Override
+        public boolean needsToProcessRenditionsForDocument(DocumentId documentId) {
+            // not needed
+            return false;
+        }
+    }
+
+}
+```
+
+You will need the following maven dependencies:
+```
+<properties>
+    <!-- properties were taken from 10.3.x -->
+    <!-- see https://github.com/nuxeo/nuxeo-arender-connector/wiki/Release-Matrix -->
```

Comment:
This is a private link, we can probably create something under a new NEV section next to release notes, wdyt?